### PR TITLE
Removing old blogs

### PR DESCRIPTION
**Pull request by prady0t**

### CODE REVIEW

**Comment by prady0t** [comment]
@agriyakhetarpal would love your reviews on this!

File: .DS_Store
Comment (agriyakhetarpal) [inline .DS_Store:1]: This is a binary file, could be removed and git-ignored

File: content/posts/GSoC-week-4.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+![GSoC Banner](/GSoC_Banner.png)
+<img src = "/GSoC_Banner.png">
+
+This blog will cover the seventh and eighth week as a GSoC student.
+
+### Work so far and for the coming weeks.
+
+- Moving all the integration test files to pytest : Finally all the integration test files have been migrated to pytest. With this integration we also ensure not using unittest anywhere in the integration test suite and moving to pytest ecosystem. There ere some large files that needed a bit of time as well as some which were fairly straightforward (using numpy testing so not much changes were required). To know more follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4285).
```
Comment (agriyakhetarpal) [inline content/posts/GSoC-week-4.md]:
```suggestion
- Moving all the integration test files to pytest : Finally all the integration test files have been migrated to pytest. With this integration we also ensure not using unittest anywhere in the integration test suite and moving to pytest ecosystem. There were some large files that needed a bit of time as well as some which were fairly straightforward (using numpy testing so not much changes were required). To know more follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4285).
```

File: content/posts/GSoC-week-6.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+date: 2024-09-12
+
+  
+
+---
+
+![GSoC Banner](/GSoC_Banner.png)
+<img src = "/GSoC_Banner.png">
+
+This blog will cover the ninth and tenth week as a GSoC student.
```
Comment (agriyakhetarpal) [inline content/posts/GSoC-week-6.md]:
```suggestion
This blog will cover the eleventh and twelth week as a GSoC student.
```

File: content/posts/GSoC-week-6.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+### Work so far and for the coming weeks.
+
+- Migrating tests to pytest part 6 : This PR covered the part 6 of pytest migration. This PR covers a lot of changes to the codebase, leveraging the power of pytest's fixtures and parameterization. With this we moved 18 files to pytest. Follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4354) to know more.
+
+- Migrating tests to pytest part 7 : This is the last PR for pytest migration. Almost all the files were moved to pytest. The ones remaning will be covered in a last PR. To know more follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4431).
```
Comment (agriyakhetarpal) [inline content/posts/GSoC-week-6.md]:
```suggestion
- Migrating tests to pytest part 7 : This is the last PR for pytest migration. Almost all the files were moved to pytest. The ones remaining will be covered in a last PR. To know more follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4431).
```

File: content/posts/GSoC-week-6.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+- Final pytest migration : This PR covers all the remaining file left from the previous work. With this PR we I also conclude the main aim of the project. You can follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4443) to know more.
```
Comment (agriyakhetarpal) [inline content/posts/GSoC-week-6.md]:
```suggestion
- Final pytest migration : This PR covers all the remaining file left from the previous work. With this PR we also conclude the main aim of the project. You can follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4443) to know more.
```

Should be either "we" or "I"

**Review by agriyakhetarpal — APPROVED** [review]

**Pull request merged**